docs/kube-api-custom-ocp-versions.md

```diff
@@ -0,0 +1,94 @@
+# Custom OpenShift Versions (Dynamic z-stream Support)
```

Comment:
Maybe add a reference to this page under:
https://github.com/openshift/assisted-service/blob/master/docs/Kube-API.md#clusterdeployment